core/src/main/java/io/spine/core/EventEnvelope.java

```diff
@@ -96,15 +96,23 @@ public ActorContext getActorContext() {
                                 .getActorContext();
     }
 
+    
     /**
-     * Sets the context of the enclosed event as origin for the event which context
-     * is being built.
+     * Passes data from the enclosed event to the event context being built.
```

Comment:
The previous method name highlighted the fact that the message within the envelope is the origin for the event being built. The new name lost this meaning.

The new description of the method makes me ask: ”Why does it do it?”